backends/platform/sdl/psp2/libvita2d/include/bin_packing_2d.h

```diff
@@ -0,0 +1,37 @@
+#ifndef BIN_PACKING_2D_H
```

Comment:
Since this code is removed, how it should be possible to build the port?

One of the alternatives is that you provide a pre-built package with sources and we put it to our server, and you refer to it from our Wiki at the port building instructions.